src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyGroupSizes.cs

```diff
@@ -13,9 +13,13 @@ public static IEnumerable<object[]> CurrencyGroupSizes_TestData()
         {
             yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 3 } };
             yield return new object[] { new CultureInfo("en-US").NumberFormat, new int[] { 3 } };
-            
-            if (!PlatformDetection.IsWindows7)
+
+            // TODO: when dotnet/corefx#2103 is addressed, we should also check fr-FR
+            if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610
```

Comment:
does Ubuntu, Fedora, Windows 8 not supporting this culture "ur-IN"? or it support it with different values than new int[] { 3, 2 }? 

if these platforms support this culture, it would be better to set the expected values for such platforms instead of just excluding it. otherwise, we should just remove testing this specific culture